src/Microsoft.PowerShell.Commands.Management/commands/management/GetChildrenCommand.cs

```diff
@@ -172,6 +172,25 @@ public uint Depth
         }
 
         /// <summary>
+        /// Gets or sets whether symbolic links are to be followed when recursing.
+        /// </summary>
+        [Parameter]
+        public SwitchParameter FollowSymlink
```

Comment:
Maybe it will turn out that making `-FollowSymlink` implicitly imply `-Recurse` is better, but for now here are my 2 cents:
1. It's always better to force the user to be explicit about their intent, so `dir -Recurse -FollowSymlink` is more readable than `dir -FollowSymlink` in my opinion.
2. For simplicity reason, this PR can just add the `-FollowSymlink` parameter without worrying about its dependency to `-Recurse`. In my understanding (correct me if I'm wrong) the related code won't execute when it's not recursively iterating file system items. Then we can get feedback from users and base on that to decide if we should make `-FollowSymlink` implicitly imply `-Recurse`.